packages/react-i18n/src/babel-plugin.ts

```diff
@@ -124,9 +120,17 @@ export default function injectWithI18nArguments({
               filename,
               fallbackID,
               insertImport() {
-                lastImport.insertAfter(
-                  fallbackTranslationsImport({id: fallbackID}),
-                );
+                const lastImport = program
```

Comment:
Now that I've written the PR, I'm wondering why we don't just insert after the `withI18n` path?  It seems like that'd avoid storing `program` in state.

Reply:
There is no guarantee the `withI18n` path will be in a location that you can insert an import (and there is a guarantee that `useI18n` will *not* be in a place you can insert an import. I do think it should be fine to just add it as the first line in the program, though.